hockey/indivPlayerFunctions.php

```diff
@@ -0,0 +1,63 @@
+<?php
+function playerInfo($LastName){
+    $info = dbQuery("
+    SELECT *
+    FROM hockey_1617
+    WHERE LastName= :var1",
+    array(
+        "var1"=>$LastName)
+    );
+    return $info->fetch();
+};
+
+function displayPlayerInfo ($name){
+    if($_REQUEST['name']==$name){
+        $result = playerInfo($name);
+        return $result;
+    }
+}
+function playerInfo1516($LastName){
+    $info = dbQuery("
+    SELECT *
+    FROM hockey_1516
+    WHERE LastName= :var1",
+    array(
+        "var1"=>$LastName)
+    );
+    return $info->fetch();
+};
+function displayPlayerInfo1516 ($name){
```

Comment:
What does the `1516` at the end of stuff signify? For things like that which aren't totally obvious, it can be helpful above a function like this to explain it.